packages/shared/ReactTypes.js

```diff
@@ -45,3 +45,8 @@ export type ReactPortal = {
   // TODO: figure out the API for cross-renderer implementation.
   implementation: any,
 };
+
+export type RefObject = {
+	contents: any,
```

Comment:
How can we better type this so it's not any?

Reply:
It can be generic
```js
export type RefObject<T> = {
  contents: T | null
};
```